nautobot/extras/tests/test_views.py

```diff
@@ -766,7 +766,7 @@ def setUpTestData(cls):
             "name,slug,remote_url,branch,secrets_group,provided_contents",
             "Git Repository 5,git_repo_5,https://example.com,main,,extras.configcontext",
             "Git Repository 6,git_repo_6,https://example.com,develop,Secrets Group 2,",
-            'Git Repository 7,git_repo_7,https://example.com,next,Secrets Group 2,"extras.job,extras.configcontext"',
+            'Git Repository 7,git_repo_7,https://example.com,next,Secrets Group 2,"extras.job,extras.exporttemplate"',
```

Comment:
Why this change here?

Reply:
Actually this was a bug caught by this change. Previously, since all of the rows were being created in a single operation, the `clean()` check for multiple GitRepositories with the same `remote_url` and overlapping `provided_contents` wasn't catching the conflict between "Git Repository 5" and "Git Repository 7" both providing `extras.configcontext` from `https://example.com`. With this change, that conflict is now detected and causes the import to fail, hence why I had to change the data.